### DISCUSSION TITLE
Embed Video

### DISCUSSION THREAD

**Discussion by sazovsky** [roam-garden/i#84] [Feature requests & Ideas]
It would be great if the video is displayed as embedded. Not just a link. 
![image](https://user-images.githubusercontent.com/86047632/133083757-37fbeffc-2d1a-4bad-ac3c-8c7e3e859c06.png)

**Comment by Stvad** [comment]
Yep this is in plans. In the meantime you can work around with using the html video tag (#51) 
https://canary.roam.garden/ has an example of video embedded that way
